src/platform/sdl2-platform.c

```diff
@@ -276,6 +276,7 @@ static boolean pollBrogueEvent(rogueEvent *returnEvent, boolean textInput) {
                 SDL_SetWindowFullscreen(Win,
                     (SDL_GetWindowFlags(Win) & SDL_WINDOW_FULLSCREEN_DESKTOP) ? 0 : SDL_WINDOW_FULLSCREEN_DESKTOP);
                 refreshWindow();
+                return false;
```

Comment:
What do you think of having this as `continue` instead? I wonder if it would work, but it seems more natural, to skip this key event in the queue but continue processing, rather than return that there are no valid events early

Reply:
You're right. `continue` seems to be the better way here.